lib/outputmanager.js

```diff
@@ -234,6 +234,11 @@ OutputPlugin.prototype.sanitizeCourseJSON = function(mode, json, next) {
       }
     }
 
+    // _themePreset should not be included in export
+    if (mode === Constants.Modes.Export) {
```

Comment:
Question: why solely not in export? It's included in publishes at present.

Reply:
I wish I could remember the answer, my fault for not documenting a reason at the time.. 

I will test removing it in publishes and hopefully remove from there also.